README.md

```diff
@@ -19,15 +19,19 @@ Thank you for your interest in contributing to the Login.gov IdP! For complete i
 
 This installation method is meant for those who are familiar with setting up local development environments on their machines.
 
+We recommend using [Homebrew](https://brew.sh/), [rbenv](https://github.com/rbenv/rbenv), [nvm](https://github.com/nvm-sh/nvm) or other version management tooling to install the below dependencies; while we don't anticipate changing these frequently, this will ensure that you will be able to easily switch to different versions as needed...
```

Comment:
Style-wise, I would drop the trailing ellipses

```suggestion
We recommend using [Homebrew](https://brew.sh/), [rbenv](https://github.com/rbenv/rbenv), [nvm](https://github.com/nvm-sh/nvm) or other version management tooling to install the below dependencies; while we don't anticipate changing these frequently, this will ensure that you will be able to easily switch to different versions as needed.
```